components/select_team/select_team.tsx

```diff
@@ -176,8 +180,18 @@ export default class SelectTeam extends React.PureComponent<Props, State> {
             canJoinPublicTeams,
             canJoinPrivateTeams,
             totalTeamsCount,
+            isCloud,
+            isCloudFreeEnabled,
+            usageDeltas: {
+                teams: {
+                    active: usageDeltaTeams,
+                },
+            },
         } = this.props;
 
+        const teamsLimitReached = usageDeltaTeams >= 0;
```

Comment:
If not cloud, does the usage delta return with no error?

Reply:
Correct, it has backup values.